boilerplate/frontend/react-multitenancy/src/Home/index.tsx

```diff
@@ -0,0 +1,26 @@
+export default function Home() {
+    return (
+        <div className="fill" id="home-container">
+            <div className="logos">
+                <img src="/ST.svg" alt="SuperTokens" />
+                <span>x</span>
+                <img src="/React.svg" alt="React" />
+            </div>
+            <div className="main-container">
+                <div className="inner-content">
+                    <p>
+                        <strong>SuperTokens</strong> x <strong>React</strong> <br /> example project
+                    </p>
+                    <div className="buttons">
+                        <a href="/auth" className="sessionButton">
+                            Sign-up / Login
+                        </a>
+                        <a href="/dashboard" className="sessionButton">
+                            Dashboard
+                        </a>
```

Comment:
This is inconsitent with the latest changes in [this PR](https://github.com/supertokens/create-supertokens-app/pull/132/files#)

Reply:
Apologies, I missed a couple things from the refactor and didn't realize. I pushed up a change for this but I need to re-fix the css and move some logic around to match. I'll get this done today or tomorrow

Reply:
This specific error is fixed